setup/setup.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+import { setupStructure as setupDb } from '../app/db/structure';
+import { setupStructure as setupS3 } from '../app/s3/structure';
+import { addData } from './fixtures/fixtures';
+
+const arg = (a) => process.argv.indexOf(a) !== -1;
+var fns = [];
+
+if (arg('--data')) {
+  fns.push(() => setupDb());
+  fns.push(() => addData());
+} else if (arg('--db')) {
+  fns.push(() => setupDb());
+}
+
+if (arg('--s3')) {
+  fns.push(() => setupS3());
+}
+
+// No specifics. Do not setup data.
+if (!fns.length) {
+  fns.push(() => setupDb());
+  fns.push(() => setupS3());
```

Comment:
No, if you don't pass any flags it's assumed you want to do the full production setup, excluding dummy data.